app/code/Magento/MediaGalleryApi/Api/Data/AssetInterface.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Asset Interface
+ *
+ */
+interface AssetInterface extends ExtensibleDataInterface
```

Comment:
Could you add a description of interface purposes? Also, could you update a description of other API interfaces?

Reply:
Done.